src/drivers/net/bnxt/bnxt.c

```diff
@@ -2116,8 +2134,8 @@ int bnxt_hwrm_run ( hwrm_func_t cmds[], struct bnxt *bp )
 	int ret;
 
 	for ( ptr = cmds; *ptr; ++ptr ) {
-		memset ( bp->hwrm_addr_req,  0, REQ_BUFFER_SIZE );
-		memset ( bp->hwrm_addr_resp, 0, RESP_BUFFER_SIZE );
+		memset ( ( void * )bp->req_addr_mapping,  0, REQ_BUFFER_SIZE );
```

Comment:
Doesn't necessarily need to be changed, but an explicit cast to `void *` is not necessary.